dm/master/status.go

```diff
@@ -39,6 +40,12 @@ func (h *statusHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 func InitStatus(lis net.Listener) {
 	mux := http.NewServeMux()
 	mux.Handle("/status", &statusHandler{})
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
Why we need to handle these four separately ?

Reply:
These four are not builtin profile, and their handler must be added to url route if we want to use them.